src/utils/Utils.java

```diff
@@ -13,9 +16,10 @@
 
 public class Utils {
 
-	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation, Double sceneWidth, Double sceneHeight) {
+	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation, String bundleLocation, Double sceneWidth, Double sceneHeight) {
         try {
-        	FXMLLoader fxmlLoader = new FXMLLoader(parent.getClass().getResource(fxmlLocation));
+        	ResourceBundle bundle = ResourceBundle.getBundle(bundleLocation);
+          	FXMLLoader fxmlLoader = new FXMLLoader(parent.getClass().getResource(fxmlLocation), bundle);
```

Comment:
If we send `bundleLocation` as null this may crash, lets change line 21 and 22 like this:
```
URL fxmlUrl = parent.getClass().getResource(fxmlLocation);
FXMLLoader fxmlLoader = bundleLocation != null ? bundleLocation new FXMLLoader(fxmlUrl, ResourceBundle.getBundle(bundleLocation)) : new FXMLLoader(fxmlUrl);
```